src/main/scala/io/citrine/lolo/transformers/standardizer/StandardizerPrediction.scala

```diff
@@ -0,0 +1,64 @@
+package io.citrine.lolo.transformers.standardizer
+
+import io.citrine.lolo.PredictionResult
+
+trait StandardizerPrediction[+T] extends PredictionResult[T] {
+
+  /** The base prediction from the model trained on standardized data. */
+  def basePrediction: PredictionResult[T]
+}
+
+case class RegressionStandardizerPrediction(
+    basePrediction: PredictionResult[Double],
+    outputTrans: Standardization,
+    inputTrans: Seq[Option[Standardization]]
+) extends StandardizerPrediction[Double] {
+
+  override def getExpected(): Seq[Double] = basePrediction.getExpected().map(outputTrans.invert)
+
+  // TODO: A PredictionResult[Double] should always return a Option[Seq[Double]] for uncertainty
+  override def getUncertainty(includeNoise: Boolean = true): Option[Seq[Any]] = {
+    basePrediction.getUncertainty(includeNoise).map { x =>
+      x.map(_.asInstanceOf[Double] * outputTrans.scale)
+    }
+  }
+
+  override def getGradient(): Option[Seq[Vector[Double]]] = {
```

Comment:
Unfortunately these are in new files and not picked up by the git diff. I did make some changes in how the `Option` is handled when implementing a handful of these methods, but the logic is intended to remain the same.

I think the `Standardizer` tests passing unchanged is a good hint that we didn't make a mistake here.